include/sysdolphin/baselib/object.h

```diff
@@ -30,6 +30,26 @@ typedef enum _HSD_Type {
     HSD_MAX_TYPE,
 } HSD_Type;
 
+typedef enum _HSD_TypeMask {
```

Comment:
Would rather see a macro here
```c
#define MASK_OF(ordinal) (1 << (ordinal-1))
// ...
AOBJ_BITS = MASK_OF(AOBJ_TYPE)
```
and then probably rename `*_BITS` to `*_MASK` to match `HSD_TypeMask`